rosbag2_cpp/src/rosbag2_cpp/service_utils.cpp

```diff
@@ -132,6 +132,8 @@ size_t get_serialization_size_for_service_metadata_event()
     static_cast<const rosidl_typesupport_introspection_cpp::MessageMembers *>(
     type_support_handle->data);
 
+  // TODO(morlov): We shall not rely on this arithmetic!!! It is up to the serialization
+  //  implementation
```

Comment:
trying make sure it can find either of us by github handle names.

```suggestion
  // TODO(@MichaelOrlov @fujitatomoya @Barry-Xu-2018 )
  // We shall not rely on this arithmetic!!! It is up to the serialization implementation.
```

Reply:
@fujitatomoya I am going to delete this code and put a comment on why it is not acceptable to go with it.
Sorry, I didn't have time to add a comment about it in the review.